### PR TITLE
Allow summing/averaging of DurationFields

### PR DESCRIPTION
**Pull request by me2d09**
Add DurationType in order to allow summing/averaging of DurationFields. 
In addition it fixes Date conversion to naive when date was already in naive.

For MySQL backend it depends on this fix: https://github.com/django/django/pull/13398

### CODE REVIEW

**Comment by tolomea** [comment]
Thank you, I will take a look at this this evening.
I will also take a look at why actions aren't running for pull requests :)

**Comment by tolomea** [comment]
Did you get the tests running locally? Given you added a type I imagine they are failing on the snapshot stuff.

**Comment by me2d09** [comment]
First of all - thank you, this repo is incredible piece of work and it is working great. About tests - no I didn't run them because I thought they are broken. I fixed the problem (see last commit) and tests are working with Django 3.1 and SQLite with Json1 extension. Now I will fix snapshots tests.

**Comment by tolomea** [comment]
I'm also looking at it. Given where I'm at it might be easier if I finish it.

File: data_browser/orm_admin.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
     ArrayField = None.__class__
 
 try:
-    from django.fields import JSONField
```
Comment (tolomea) [inline data_browser/orm_admin.py:46]: this is a fascinating failure on my part that raises all sorts of questions

Comment (tolomea) [reply data_browser/orm_admin.py:46]: oh this works for me on 3.1 because I have the postgres library installed
but why does it pass on CI

Comment (me2d09) [reply data_browser/orm_admin.py:46]: I figured it out because I have no postgresql installed and almost all tests failed.

**Comment by me2d09** [comment]
> 
> 
> I'm also looking at it. Given where I'm at it might be easier if I finish it.

Ok, thanks!

**Comment by tolomea** [comment]
Could you enable this
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests
so I can add my changes onto your pull request.

**Comment by me2d09** [comment]
> 
> Could you enable this ...

It is enabled from beginning, so it should work...

**Comment by me2d09** [comment]
image

**Comment by tolomea** [comment]
Ok, thanks, I haven't done this flow before so I'm probably doing it wrong,
I'll figure it out.
The mysql thing is annoying it'd be good to find a workaround.
Incidentally if you are using mysql and haven't come across it yet my
friend has a useful package https://django-mysql.readthedocs.io/en/latest/

File: data_browser/orm_admin.py (hunk 1 above)
Comment (tolomea) [reply data_browser/orm_admin.py:46]: Thanks for that, I would not have noticed until someone bug reported it.

**Comment by tolomea** [comment]
There's a work around.
Thank you for the pull request.

**Pull request merged**